devicetypes/smartthings/zigbee-metering-plug.src/zigbee-metering-plug.groovy

```diff
@@ -162,7 +176,12 @@ private int getPowerDiv() {
 }
 
 private int getEnergyDiv() {
-    isSengledOutlet() ? 10000 : 100
+    if (device.getDataValue("manufacturer") == "DAWON_DNS") {

```

Comment:
Please change the code as @PKacprowiczS suggested.

Reply:
I changed the code as below. Did I get it right?

private int getEnergyDiv() {
    if (isDawonOutlet()) {
        1000
    } else if (isSengledOutlet()) {
        10000
    } else {
        100
    }
}

private boolean isDawonOutlet() {
    device.getDataValue("manufacturer") == "DAWON_DNS"
}

Reply:
Yeah, just remember about indentation and it should be fine.